platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxEventWrapper.java

```diff
@@ -21,24 +22,32 @@
   static Hashtable<String, Object> buildMapClickEvent(
     @NonNull Location location, @NonNull String gestureId, Transform transform) {
     try {
-      return MapboxEvent.buildMapClickEvent(location, gestureId, transform.getZoom());
+      double mapZoom = transform.getZoom();
+      if (mapZoom >= MapboxConstants.MAXIMUM_ZOOM && mapZoom <= MapboxConstants.MAXIMUM_ZOOM) {
```

Comment:
We should use `MapboxConstants.MINIMUM_ZOOM` in the first check. Typo? ;)

Reply:
thanks for catching that!